src/client/components/AddReservation.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { useState, useEffect } from "react";
+import postData from "./postData";
+import Meals from "./Meals";
+
+export default function AddReservation(reservationId) {
```

Comment:
nice work in here :)

Reply:
Thank you so much for your reviews. I will implement it soon 👍